packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -213,15 +194,14 @@ ApiReportGenerator.generateReviewFileContent =
     );
 
     try {
-      const prettier = require('prettier') as typeof prettierType;
+      const prettier = require('prettier') as typeof import('prettier');
 
       const config = prettier.resolveConfig.sync(cliPaths.targetRoot) ?? {};
       return prettier.format(content, {
         ...config,
         parser: 'markdown',
       });
     } catch (e) {
```

Comment:
will try to raise this on the AE project, maybe makes sense for them to add a `beforeWrite` hook

Reply:
Yeah that could be neat